apps/fluent-tester/src/FluentTester/TestComponents/Avatar/AvatarTest.tsx

```diff
@@ -1,18 +1,30 @@
+};
```

Comment:
Because this is so large, we could probably move this to it's own TS file. We want base64 instead of say, a PNG because I was trying to explicitly test base64 support